test/modules/home/home_view_test.dart

```diff
@@ -201,6 +219,7 @@ void main() {
         module.view.beginSkeletonAnimation();
         await tester.pump();
         await tester.pump();
+        await tester.pump();
```

Comment:
we need another pump? 😳

Reply:
Yes, after wrapping it with `SliderMenuContainer`, I have to add one more `pump` to make it works.. 😓